### PR TITLE
libstrophe 0.8.8

### PR DESCRIPTION
**Pull request by hanskuhn**
Simple version bump. First time contribution so please let me know if I've made any mistakes. I followed the commit process at https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md

### CODE REVIEW

File: Library/Formula/libstrophe.rb [hunk 1]
```diff
@@ -10,9 +10,9 @@ class Libstrophe < Formula
     cellar :any
     revision 2
     sha256 "b533ba5fbc93054ff666907002045eb057decaa582223610135bf5240f2f579b" => :el_capitan
-    sha256 "2ae7ce796b2f9318c7dd156c9014fcd214ab88d28e9af4b081161fc2166d732e" => :yosemite
-    sha256 "6bd460eafc013874c19fc20734d82a43deba034da23df9bf8e54533b79740686" => :mavericks
-    sha256 "64de9bbabe8ff8fd860d3db9d57eb55a83e5899f469731751f7da613a9155752" => :mountain_lion
+    sha1 "cba04617714a99e25d71ab0efd9d26d046d336c4" => :yosemite
+    sha1 "1f824c9bcb4d8e370e5423d26b75bc926de23afc" => :mavericks
+    sha1 "5ed9df92858ba19d256cfc99c1459649b4840f68" => :mountain_lion
   end
```
Comment (bfontaine) [inline Library/Formula/libstrophe.rb]: Please leave the `bottle do` block as is; our bot will update it when pulling. Thanks!

Comment (hanskuhn) [inline Library/Formula/libstrophe.rb]: Thanks for the feedback! Does this note require I re-submit with the original bottle lines intact? Or is this just for future submits?

File: Library/Formula/libstrophe.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class Libstrophe < Formula
   desc "XMPP library for C"
   homepage "http://strophe.im/libstrophe/"
-  url "https://github.com/strophe/libstrophe/archive/0.8.6.tar.gz"
-  sha256 "a2158134381e544a7697e9379f45abe18da817cd05eb3272eabec2151599d702"
+  url "https://github.com/strophe/libstrophe/archive/0.8.8.tar.gz"
+  sha256 "08f4a85ef419a8bdf08b6afa8f7b2a0e5e180fdc9c16cede81af672ec10e21e7"
   head "https://github.com/strophe/libstrophe.git"
   revision 1
```
Comment (DomT4) [inline Library/Formula/libstrophe.rb]: Please remove any `revision` line *outside* the `bottle do` block from the formula on stable version updates, Thanks!

File: Library/Formula/libstrophe.rb (hunk 1 above)
Comment (mikemcquaid) [inline Library/Formula/libstrophe.rb]: You can just push a new commit (or squash into the existing commit and force push)  into this commit, thanks.

**Comment by hanskuhn** [comment]
Sorry about the mess with the commits. I was having problems with rebasing b/c the EDITOR variable was poorly chosen (remote display) and it took a while to figure out what I was doing wrong.

 f82e2d1 looks like a winner now with all feedback integrated.

**Comment by bfontaine** [comment]
Thank you for your (first) contribution to Homebrew @hanskuhn; we appreciate it! :tada:

**Comment by hanskuhn** [comment]
My pleasure @bfontaine. Thanks to all of you for helping me move this forward!

**Pull request closed without merging**